### DISCUSSION TITLE
dev版本升级了一下，一直提示fething data，无奈又重新部署了一下。

### DISCUSSION THREAD

**Discussion by yyysuo** [usememos/memos#62] [General]
创建命令是：
docker run -d --name memos --restart=unless-stopped --mount type=tmpfs,destination=/tmp -p 5230:8080 -v /mnt/sda3/memos/:/var/opt/memos -e mode=prod neosmemo/memos:dev

大佬给指导一下呗。另外memos配合Iphone的快捷指令真是太好用了，不仅可以方案的写日记，还可以直接在锁屏状态下用语音命令直接添加日记。感谢！

**Comment by boojack** [comment]
因为目前还处于开发中，如果出现这种情况，估计是 schema 变更后的问题。
建议先从旧版本导出数据，再重新启动新版本，不要 mount 到旧的数据库文件，让它自己生成新的数据库，然后导入数据。
<img width="710" alt="image" src="https://user-images.githubusercontent.com/24653555/169428353-d5c8de46-0577-4e6f-a4ae-4e6cf8158708.png">

**Reply by yyysuo** [reply]
> 

非常感谢，另外，mode参数是什么意思呢？我看之前有用release的。

**Reply by boojack** [reply]
`mode: "dev" | "prod"`
https://github.com/justmemos/memos/blob/main/common/profile.go#L13
https://github.com/justmemos/memos/blob/main/store/sqlite.go#L57

**Reply by yyysuo** [reply]
> `mode: "dev" | "prod"` https://github.com/justmemos/memos/blob/main/common/profile.go#L13 https://github.com/justmemos/memos/blob/main/store/sqlite.go#L57

多谢，明白了，一个开发，一个生产。
